### ISSUE TITLE
Tabs component tab index not functional / accessibility issue [SCREENCAPS ATTACHED]

### ISSUE THREAD

**Issue by Haeven** [shadcn-ui/ui#2222]
Tab indexing (a core accessibility feature) is currently not functional for the Tabs component, specifically you cannot tab into inactive tabs as depicted in the following videos. I really enjoy using this library and resonate strongly with the values of the creator, so it would be my pleasure to submit a PR for this 😄 

[SCREEN CAPTURES]
https://github.com/shadcn-ui/ui/assets/42015067/b6326c56-40a7-43fa-8273-96d82033176a

https://github.com/shadcn-ui/ui/assets/42015067/fd572de0-5246-458e-8254-6cdc9b4a464e

**Comment by Haeven** [comment]
You can use the keyboard to index between inactive tabs

**Comment by 58bits** [comment]
We've also just come across this - and have seen that you can use the left and right arrows to move between all tabs. Does this satisfy accessibility requirements?
